Neos.Flow/Classes/Command/ConfigurationCommandController.php

```diff
@@ -64,16 +64,18 @@ class ConfigurationCommandController extends CommandController
      *
      * @param string $type Configuration type to show, defaults to Settings
      * @param string $path path to subconfiguration separated by "." like "Neos.Flow"
+     * @param int $level Truncate the configuration at this depth and show '...'
      * @return void
      */
-    public function showCommand(string $type = 'Settings', string $path = null)
+    public function showCommand(string $type = 'Settings', string $path = null, int $level = 0)
```

Comment:
Level suggests to me that it somehow _starts_ at that level? Maybe truncateAfter?

Reply:
hmm true, i did it similar to https://github.com/neos/neos-development-collection/pull/4619 to be consistent

Reply:
so id keep at is okay?

Reply:
as this is the level of how much will be shown and where its truncated.

Reply:
Or potentially `--depth`

Reply:
jup exactly ... its like our `Neos.Flow.error.debugger.recursionLimit` that was never a recursion limit to begin with ... it just ensures the output is untouched until this array nesting and will truncate everything else as shown above.